src/Services/TupleFilterService.php

```
@@ -102,18 +102,37 @@
+            }
+
+            $conditionKey = '#' . md5($encoded);
+        }
+
         return sprintf(
             '%s#%s@%s%s',
-            $tuple->getUser() ?? '',
-            $tuple->getRelation() ?? '',
-            $tuple->getObject() ?? '',
-            $condition instanceof ConditionInterface ? '#' . spl_object_hash($condition) : '',
+return sprintf(
```

PHPStan (2.1.15)

Syntax error, unexpected T_STRING, expecting ':' on line 129


(phpstan.parse)